core-client/transports/src/transports/mod.rs

```diff
@@ -81,7 +81,7 @@ impl RequestBuilder {
 pub fn parse_response(
 	response: &str,
 ) -> Result<(Id, Result<Value, RpcError>, Option<String>, Option<SubscriptionId>), RpcError> {
-	serde_json::from_str::<ClientResponse>(&response)
+	jsonrpc_core::serde_from_str::<ClientResponse>(response)
```

Comment:
Can we now get rid of `serde_json` dependency?

Reply:
`serde_json` is still used for all `to_string` serializations